src/client/datascience/baseJupyterSession.ts

```diff
@@ -282,13 +281,7 @@ export abstract class BaseJupyterSession implements IJupyterSession {
     }
 
     // Sub classes need to implement their own restarting specific code
-    protected abstract startRestartSession(timeout: number): void;
-    // Sub classes need to implement their own kernel change specific code
-    protected abstract createNewKernelSession(
```

Comment:
This method isn't called from sub class, only ever class in jupyter class (method was never used in raw kernels)